alfred-telemetry-solr/build.gradle

```diff
@@ -13,10 +13,43 @@ subprojects {
 
     apply from: "${project.projectDir}/overload.gradle"
 
+    description = "Alfred Telemetry Solr extension for ${solrFlavor} which enables exposure of Metrics using Micrometer"
+
     plugins.withId('java') {
+        configurations {
+            implementationSources {
```

Comment:
I understand this a configuration specific for dependencies only required for Javadoc?

Reply:
Yes, this uses gradle attribute matching to pull in the sources jar of `alfred-telemetry-solr-common` and then merge that one into the javadoc and sources jars.
I want those jars to be correct, because it helps a lot when debugging in intellij.
Sonatype requires a sources and javadoc jar, although they are not validated that they cover all compiled code, it's good to have